src/install.js

```diff
@@ -209,32 +209,26 @@ Run ppm -v after installing Git to see what version has been detected.\
     // packageName - The string name of the package to request.
     //
     // return value - A Promise that rejects with an appropriate error or resolves to the response body
-    requestPackage(packageName) {
+    async requestPackage(packageName) {
       const requestSettings = {
         url: `${config.getAtomPackagesUrl()}/${packageName}`,
         json: true,
         retries: 4
       };
-      return new Promise((resolve, reject) => {
-        request.get(requestSettings, (error, response, body) => {
-          let message;
-          body ??= {};
-          if (error != null) {
-            message = `Request for package information failed: ${error.message}`;
-            if (error.status) { message += ` (${error.status})`; }
-            return void reject(message);
-          }
-          if (response.statusCode !== 200) {
-            message = request.getErrorMessage(body, error);
-            return void reject(`Request for package information failed: ${message}`);
-          }
-          if (!body.releases.latest) {
-            return void reject(`No releases available for ${packageName}`);
-          }
-
-          resolve(body);
-        });
+      const response = await request.get(requestSettings).catch(error => {
+        const message = request.getErrorMessage(body, error);
```

Comment:
The first parameter here `body` doesn't exist, as we are only inside the `catch()` callback.

Reply:
I feel this was "too clever". I rather don't want to change the logic here at the moment, I will revert to the original way of producing the error message here.